lib/puppet/configurer.rb

```diff
@@ -92,7 +104,9 @@ def get_facts(options)
       # get a hash with both of these pieces of information.
       #
       # facts_for_uploading may set Puppet[:node_name_value] as a side effect
-      facts_hash = facts_for_uploading
+      facts_hast = nil
```

Comment:
should be `facts_hash`, but it's already initialized above, so initializing to `nil` isn't necessary.